src/main/java/by/beloboky/countAllLetterFromFiles/FileRepository.java

```diff
@@ -0,0 +1,27 @@
+package by.beloboky.countAllLetterFromFiles;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.Stream;
+
+public class FileRepository {
+    private final Path path;
+
+    public FileRepository() {
+        this.path = Paths.get("/home/andrey/Desktop/Task/multithreading deadlock/files");
+    }
+
+    /**
+     * @return - List<String> which consists(includes) name of the 50th files.
+     */
+    public List<String> findFiles() {
```

Comment:
If you would like to return file Path, than it is better to return List of Path. Why you map it to the string?